### PR TITLE
Make proxy.pac compatible with Safari

### PR DESCRIPTION
**Pull request by ericboehs**
Safari didn't like the regex domain matching.

Also, it wanted `SOCKS` instead of `SOCKS5` (via [SO](https://apple.stackexchange.com/a/396773)).

### CODE REVIEW

File: scripts/socks/proxy.pac [hunk 1]
```diff
@@ -1,9 +1,14 @@
 function FindProxyForURL(url, host) {
-  PROXY = "SOCKS5 127.0.0.1:2001"
+  PROXY = "SOCKS 127.0.0.1:2001"
```
Comment (ericboehs) [inline scripts/socks/proxy.pac:2]:
```suggestion
  PROXY = "SOCKS5 127.0.0.1:2001;SOCKS 127.0.0.1:2001"
```

**Comment by karlbrown-va** [comment]
Have you confirmed this still works correctly with Chrome and Firefox?

Also, you'll need to update the readme, as it says that Safari doesn't work.

thanks!

**Comment by ericboehs** [comment]
@karlbrown-va Looks like Chrome/Firefox needed `SOCKS5`. I've updated the `PROXY` to include both and from my testings on macOS 10.15.7, it works with Chrome 85 & 86, Firefox 81, and Safari 14.

**Review by karlbrown-va — APPROVED** [review]
ok, I tested this and it works for me as well.

**Pull request merged**